metron-stellar/stellar-common/README.md

```diff
@@ -702,9 +702,9 @@ Where:
   * Returns: True if the string is a valid URL and false if otherwise.
 
 ### `JOIN`
```

Comment:
Do we explicitly want to say `Java Iterable` here?

Reply:
Seems reasonable, done.